cmd/containerboot/main.go

```diff
@@ -36,9 +36,15 @@
 //   - TS_SOCKET: the path where the tailscaled LocalAPI socket should
 //     be created.
 //   - TS_AUTH_ONCE: if true, only attempt to log in if not already
-//     logged in. If false (the default, for backwards
-//     compatibility), forcibly log in every time the
-//     container starts.
+//     logged in. If false, forcibly log in every time the container starts.
+//     The default until 1.50.0 was false, but that was misleading: until
+//     1.50, containerbook used `tailscale up` which would ignore an authkey
```

Comment:
Boot

Reply:
I've been doing so much in German recently I'm surprised this didn't come out as Containerbuch.